### ISSUE TITLE
Overrides tab might not be working?

### ISSUE THREAD

**Issue by navv1234** [simulationcraft/simc#2438]
Originally reported on Google Code with ID 2439

```
IF RUNNING FROM THE GUI, PLEASE ATTACH (or Cut-N-Paste contents) OF
simc_gui.simc FILE.  THIS WILL ENABLE TO US TO REPRODUCE THE PROBLEM.

What steps will reproduce the problem?
1.
When trying to use the Override tab and the SpellData documentation at https://code.google.com/p/simulationcraft/wiki/SpellData
various entries do not work at all and cause "simulation setup errors" and thus the
simulation fails.

Example:  Loading the Ele shaman profile then going into the Override tab and entering
various versions of override.spell_data=spell.51505.sp_coefficient=2 (Lava Burst in
this case ought to be modified)
and similar versions like taking off the sp_.  None of the variations I've used result
in a successful sim.


What is the expected output? What do you see instead?
Expected output is the sim will scale a spell's SP coefficient and thus the change
would be notable.  Instead, the simulation entirely fails and returns the log
"Simulation setup error: Unknown option 'override.spell_data' with value 'spell.51505.sp_coefficient=2'
Simulation failed!"

When using the exact notation in the wiki page for a Frost DK which goes override.spell_data=effect.43156.base_value=100
  spell_query=spec_spell.name=icy_talons
it returns
"Option parsing error: Unexpected parameter 'simc'. Expected format: name=value
Simulation failed!"

What version of the product are you using? On what operating system?
603-26 Vista

Please provide any additional information below.

Basically just trying to get a way to change spells or abilities and compare results
between simulations without having to directly modify the code.  What is the correct
syntax for this sort of thing?  Is there coding issues, is the wiki page incorrect,
or is something else going on?
```

Reported by `thegreenjackalope` on 2015-02-13 05:28:10
